pandas/tests/scalar/test_timestamp.py

```diff
@@ -582,8 +582,9 @@ def test_pprint(self):
  'foo': 1}"""
         assert result == expected
 
-    def to_datetime_depr(self):
+    def test_to_datetime_depr(self):
         # see gh-8254
+        assert False
```

Comment:
what is the issue here? I guess this test wasn't turned on before.

Reply:
Woops, yah.  That was an earlier check that these were not getting hit.  Changing now.